DOCS-4011: Update manage-viam-server (#4334)
viamrobotics/docs: docs/operate/reference/viam-server/manage-viam-server.md

sudo rm /etc/viam.json
```

{{< alert title="Caution" color="caution" >}}

> If you remove the machine cloud credentials file you will not be able to connect to your machine without restoring it.
> You can only restore this file if you have access to the machine in the Viam app.

Reviewer: ```suggestion
If you remove the machine cloud credentials file you will not be able to connect to your machine.
You can only restore this file if you have access to the machine configuration in the Viam app.
```
same as above
